test/test_functions

```diff
@@ -267,8 +269,12 @@ cvmfs_umount() {
     sudo umount /cvmfs/$r
     local result=$?
     if [ $result -ne 0 ]; then
-      lsof /cvmfs/$r
-      return 200
+      sudo umount -l /cvmfs/$r
```

Comment:
That's dangerous because the fuse process might stay alive, which results in unexpected behavior for the next mount.  Can we revert this?